# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/shaderc/hotfix-no-3rd-party.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- shaderc-2024.0/CMakeLists.txt.vanilla	2024-04-22 19:16:25.822000000 +0200
+++ shaderc-2024.0/CMakeLists.txt	2024-04-22 19:16:32.969000000 +0200
@@ -120,7 +120,7 @@
 
 # Configure subdirectories.
 # We depend on these for later projects, so they should come first.
-add_subdirectory(third_party)
+#add_subdirectory(third_party)
 
 add_subdirectory(libshaderc_util)
 add_subdirectory(libshaderc)
